inc/scroom/viewinterface.hh

```diff
@@ -7,14 +7,156 @@
 
 #pragma once
 
+#include <stdlib.h>
+
 #include <string>
+#include <cmath>
 
 #include <gtk/gtk.h>
 
 #include <boost/shared_ptr.hpp>
 #include <boost/weak_ptr.hpp>
 
 #include <scroom/progressinterface.hh>
+#include <scroom/stuff.hh>
+#include <scroom/rectangle.hh>
+
+/**
+ * Forward declaration for ViewInterface::getCurrentPresentation()
+ */
+class PresentationInterface;
```

Comment:
I propose to get rid of comments on forward declarations.
- The declaration itself is clear enough
- Because of the `/**`, It'll end up n the doxygen api documentation, which is definitely not desirable